time stamp
uint32 image_height
uint32 image_width
uvdar_robofly/Point2DWithFloat[] points
